### PR TITLE
Update jobs.md

### PR DESCRIPTION
**Pull request by aydun**
Clarifications aimed at helping new admins

### CODE REVIEW

File: docs/setup/jobs.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ Notes:
 ```example
 /full/path/to/php /full/path/to/cv api job.execute --user=admin --cwd=/var/www/example.org
 ```
+* Replace ```cronuser``` with the CMS user you choose in the previous step.
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
* Replace ```cronuser``` with the CMS user you chose in the previous step.
```

File: docs/setup/jobs.md [hunk 2]
```diff
@@ -87,22 +88,23 @@ Notes:
...
-  -u myusername \
+  -u cronuser \
   civicrm-api job.execute
 ```
 
 Notes:
 
 * `drush` only works for Drupal sites.
 * See [drush.org](http://www.drush.org/) for more info about this tool.
+* Replace ```cronuser``` with the CMS user you choose in the previous step.
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
* Replace ```cronuser``` with the CMS user you chose in the previous step.
```

File: docs/setup/jobs.md [hunk 3]
```diff
@@ -112,6 +114,7 @@ Notes:
 
 * `wp-cli` only works for WordPress
 * See [wp-cli.org](http://wp-cli.org/) for more info about this tool.
+* Replace ```cronuser``` with the CMS user you choose in the previous step.
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
* Replace ```cronuser``` with the CMS user you chose in the previous step.
```

File: docs/setup/jobs.md [hunk 4]
```diff
@@ -120,15 +123,16 @@ Notes:
...
+  -u cronuser \
+  -p cronuserpassword \
   -e Job \
   -a execute
 ```
 
 Notes:
 
 * The `-s` parameter is the site name, and defaults to `localhost`. It should be set to the FQDN (fully qualified domain name) of your site (eg, `example.org`). Some jobs rely on this being set (in certain cases, the CiviMail mailing scheduler uses this FQDN to generate absolute URLs when relative URLs are in the email).
+* Replace ```cronuser``` and ```cronuserpassword``` with the CMS user and password you choose in the previous step.
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
* Replace ```cronuser``` and ```cronuserpassword``` with the CMS user and password you chose in the previous step.
```

File: docs/setup/jobs.md [hunk 5]
```diff
@@ -238,12 +242,21 @@ In order for the scheduled jobs configured within the CiviCRM UI to run automati
 ### crontab {:#crontab}
 
 1. Make sure you have chosen (and tested) one of the [commands to call `Job.execute`](#commands). In the following examples, we will refer to your command as `<command>`.
-1. Run the following command to edit your crontab:
+1. You need to edit the crontab of the OS User you deermined earlier.
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
1. You need to edit the crontab of the OS User you determined earlier.
```

File: docs/setup/jobs.md [hunk 6]
```diff
@@ -238,12 +242,21 @@ In order for the scheduled jobs configured within the CiviCRM UI to run automati
...
+
+   a. If you are logged in as that user, run:
 
     ```
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
    ```bash
```

File: docs/setup/jobs.md [hunk 7]
```diff
@@ -238,12 +242,21 @@ In order for the scheduled jobs configured within the CiviCRM UI to run automati
...
     $ crontab -e
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
    crontab -e
```

File: docs/setup/jobs.md [hunk 8]
```diff
@@ -238,12 +242,21 @@ In order for the scheduled jobs configured within the CiviCRM UI to run automati
...
     ```
+    
+   b. Alternatively, run:
+
+    ```
+    $ sudo crontab -e -u os_user
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
    sudo crontab -e -u os_user
```

File: docs/setup/jobs.md [hunk 9]
```diff
@@ -238,12 +242,21 @@ In order for the scheduled jobs configured within the CiviCRM UI to run automati
...
+
+   a. If you are logged in as that user, run:
 
     ```
     $ crontab -e
     ```
+    
+   b. Alternatively, run:
+
+    ```
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
    ```bash
```

**Comment by aydun** [comment]
Ah, the perils of copy/paste!

File: docs/setup/jobs.md [hunk 10]
```diff
@@ -120,15 +123,16 @@ Notes:
 ```
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
```bash
```

File: docs/setup/jobs.md [hunk 11]
```diff
@@ -120,15 +123,16 @@ Notes:
...
 $ /path/to/php /path/to/civcrm/bin/cli.php \
```
Comment (MikeyMJCO) [inline docs/setup/jobs.md]:
```suggestion
/path/to/php /path/to/civcrm/bin/cli.php \
```

**Comment by MikeyMJCO** [comment]
Thanks Aydun!

Approved and merging!

**Pull request merged**